frontend/src/component/project/Project/ProjectOverview.tsx

```diff
@@ -138,14 +146,29 @@ const ProjectOverview = () => {
 
     if (featureSearchFrontend) return <PaginatedProjectOverview />;
 
+    const featureTypeCounts = features.reduce(
```

Comment:
This is so the old component will work with refactor. This code will be removed when flag gets removed.